src/mapeo-project.js

```diff
@@ -420,6 +434,18 @@ export class MapeoProject extends TypedEmitter {
 
         if (schemaName === 'projectSettings') {
           projectSettingsEntries.push(entry)
+        } else if (schemaName === 'translation') {
+          const doc = decode(entry.block, {
+            coreDiscoveryKey: entry.key,
+            index: entry.index,
+          })
+
+          // this is so that I can cast the doc as a `TranslationValue`
+          // without needing to decode the doc beforehand
+          if (doc.schemaName === 'translation') {
+            this.#translationApi.index(doc)
+          }
```

Comment:
1. Don't we decode it on the lines above?
2. Shouldn't the schema name always be `translation`?

Could we import `assert` from `utils.js` and do this?

```suggestion
          assert(doc.schemaName === 'translation', 'expected a translation doc')
          this.#translationApi.index(doc)
```